src/doc/src/reference/features.md

```diff
@@ -513,3 +513,9 @@ source and inspect it.
 
 [`cargo vendor`]: ../commands/cargo-vendor.md
 [cargo-clone-crate]: https://crates.io/crates/cargo-clone-crate
+
+### Caveats
+
+Features, which are a form of conditional compilation, require an exponential number of configurations and test cases to be 100% covered. By default, testing, linting, Miri, docs and others will only run on the default set of features.
```

Comment:
Can you break lines on sentences?

I might reword this first sentence a little:

```suggestion
Because features are a form of conditional compilation, they require an exponential number of configurations and test cases to be 100% covered. By default, testing, linting, Miri, docs and others will only run on the default set of features.
```

As for the second sentence, I feel like it is introducing terms and tools that aren't commonly used in our documentation.  Miri is an experiment and not something I would expect most people to know what it is.  Perhaps it could be something like:

> By default, tests, docs, and other tooling such as [Clippy](https://github.com/rust-lang/rust-clippy) will only run with the default set of features.

Reply:
Done